test/e2e/suite_test.go

```diff
@@ -1933,24 +1934,47 @@ func createKeyspaceAndTable(
 ) {
 	_, err := f.ExecuteCql(ctx, k8sContext, namespace, clusterName, pod, fmt.Sprintf("CREATE KEYSPACE %s WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : %d};", keyspace, replicationFactor))
 	require.NoError(t, err, "failed to create keyspace")
-
-	_, err = f.ExecuteCql(ctx, k8sContext, namespace, clusterName, pod, fmt.Sprintf("CREATE TABLE %s.%s (id int PRIMARY KEY);", keyspace, table))
+	_, err = f.ExecuteCql(ctx, k8sContext, namespace, clusterName, pod, fmt.Sprintf("CREATE TABLE %s.%s (id INT PRIMARY KEY, v TEXT);", keyspace, table))
 	require.NoError(t, err, "failed to create table")
 }
 
+func verifyReaperSecrets(
```

Comment:
Suggestion: it would be good to check the fingerprints on the certs to make sure they're what you expect.